docs/io.cozy.files_metadata.md

```diff
@@ -502,3 +502,15 @@ to be added to a document.
   ...
 }
 ```
+
+## Toutatice shortcut files
+
+[Toutatice konnector](https://github.com/konnectors/toutatice) creates some shortcut based on application on Toutatice store. This metadata are used to organize display onto Home and Store
+
+- `title`: {string} the title of the application
+- `description`: {string} the description of the application
+- `url`: {string} the link of the application
+- `icon`: {string} the base64 icon of the application
+- `source`: {string} the space where it belongs inside Toutatice
+- `networkAccess`: {string} the rights granted to the application
+- `type`: {string} the category based onto the `source` (for example lien, info, triskell, perso)
```

Comment:
I'm confused, are they shortcuts in the same sense as https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#shortcuts ?
I see that @KillianCourvoisier added the icon and mime type recently for them: https://github.com/cozy/cozy-doctypes/pull/228/files

Reply:
Yes you are right they are shortcuts in the same sense of `io.cozy.files` with target property